### PR TITLE
rename Minetest to Luanti

### PR DESCRIPTION
**Pull request by heavygale**
Minetest got renamed to Luanti, see https://blog.minetest.net/2024/10/13/Introducing-Our-New-Name/

Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/luanti.yml [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
   - C++
+  - Lua
   - deb
 tags:
   - Games
 source_code_url: https://github.com/minetest/minetest
-stargazers_count: 10868
-updated_at: '2024-12-03'
+stargazers_count: 10870
+updated_at: '2024-12-04'
```
Comment (Ki-er) [inline software/luanti.yml:16]:
```suggestion
```

These are done automatically

**Review by Ki-er — CHANGES_REQUESTED** [review]
Will they be changing the github to Lunati too. If so I think we should hold this for a couple days until they change it over.

**Comment by heavygale** [comment]
Yes, they plan to move to https://github.com/luanti-org
The name was changed 2 month ago, I don't know how long it will take them to also move the repos to the new organization.

File: software/luanti.yml [hunk 2]
```diff
@@ -1,16 +1,15 @@
-name: Minetest
-website_url: https://www.minetest.net/
+name: Luanti
+website_url: https://www.luanti.org/
 description: An open source voxel game engine. Play one of our many games, mod a game to your liking, make your own game, or play on a multiplayer server.
```
Comment (nodiscc) [inline software/luanti.yml:3]:
```suggestion
description: An open source voxel game engine (formerly Minetest). Play one of our many games, mod a game to your liking, make your own game, or play on a multiplayer server.
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Pull request merged**